### DISCUSSION TITLE
GitHub releases for a multi module project

### DISCUSSION THREAD

**Discussion by lburgazzoli** [jreleaser/jreleaser#607] [Q&A]
I'd like to generate github releases for https://github.com/etcd-io/jetcd where I'd like to upload all the artifacts but I haven'0t found an example about how to use jreleaser in a multi module project, would it be possible to have an example ?

**Comment by aalmiray** [comment]
@lburgazzoli perhaps https://github.com/redis-developer/riot may serve as a starting point. This project used to have its JReleaser configuration defined at the root `build.gradle` file but it seems it has since migrated to using an external `jreleaser.yml` file.

My recommendation is to configure JReleaser at the root project, with a distribution per subproject. That way you can perform a release by invoking `./gradlew assemble :jreleaserFullRelease`.
